### PR TITLE
Readmes rach java7

### PR DESCRIPTION
**Pull request by Rachael-E**
A freshly cooked batch from me, covering the following. @hud10837 please could you take a first look? 

- SymbolizeShapeFile
- ReadSymbolsMobileStyleFile
- TakeScreenshot
- Terrain Exaggeration
- ArcGISTiledLayerURL

### CODE REVIEW

File: java/symbolize-shapefile/README.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
+Pan and zoom around the map to view the symbology renderer set to the feature layer created from the shapefile. 
 
 ## How it works
-1. Create a `ShapefileFeatureTable` by setting the path to a shapefile. 
-1. Create a `FeatureLayer` and set the feature layer table property to the `ShapeFileFeatureTable`.
-1. Create a `SimpleRenderer` to override the default symbology. In this case, a yellow `SimpleFillSymbol` with a red `SimpleLineSymbol` as an outline.
-1. Apply the symbol to the `FeatureLayer` by setting the renderer.
+
+1. Create a `ShapefileFeatureTable`, providing the path to a shapefile.
+2. Create a `FeatureLayer` and associate it with the `ShapeFileFeatureTable`.
```
Comment (hud10837) [inline java/symbolize-shapefile/README.md]: In similar situations the wording is usually something like "Create a `FeatureLayer` using a `ShapeFileFeatureTable`, but this isn't too hard to understand.

File: java/take-screenshot/README.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+3. On done, call `get()` on the `ListenableFuture<Bitmap>` and save it to the device.
 
 ## Relevant API
-* ArcGISMap
-* ExportImageAsync
-* ListenableFuture<Bitmap>
+
 * MapView
+* MapView.exportImageAsync
+* ListenableFuture<Bitmap>
```
Comment (hud10837) [inline java/take-screenshot/README.md]: Does specifyng bitmap make sense here? This seems out of place to me because ListenableFuture is generic.

Comment (Rachael-E) [reply java/take-screenshot/README.md]: I know what you mean and I had the same thought in my mind: I'll remove the reference.

**Comment by Rachael-E** [comment]
@hud10837 thanks for those, I've addressed them all! Since they were tiny I'll go straight to @sorenoid for a second review please.

**Pull request merged**